extensions/amp-form/0.1/test-e2e/test-ssr.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const pageWidth = 800;
```

Comment:
The default is 800x600 so you can omit specifying `initialRect` here. Same below. Sorry if you copied that from other files, I should change those

https://github.com/ampproject/amphtml/blob/e24cdde5a4d7fc00575f210741b4df323d0ca6dd/build-system/tasks/e2e/describes-e2e.js#L30

Reply:
sweet. removed.